### DISCUSSION TITLE
Implement version comparison to latest OpenFIRE GitHub release (or at least latest as of the GUI version)

### DISCUSSION THREAD

**Discussion by alessandro-satanassi** [TeamOpenFIRE/OpenFIRE-App#13] [General]
Have you by any chance already started to implement something? Or do you already have some idea of ​​how to do it? I could try to help

**Comment by SeongGino** [comment]
It is not something I considered because I don't want to have any internet dependence in the app, or would be unnecessary maintenance burden at the moment. I'll probably add it someday as there's some placeholder code for the purpose, but not now.

Also, **please** do not ask a maintainer if they're "already working on" your feature requests; that's an unnecessary bit of nagging and not courteous in the slightest.

**Reply by alessandro-satanassi** [reply]
I didn't mean to be rude and it's not my request.
It's listed in the project 'homepage' (https://github.com/TeamOpenFIRE/OpenFIRE-App/tree/v2.0-rc6) as a to-do and I thought I'd do you a favor by offering to help implement it.
See the end of the page for the text: ""TODO:
Implement version comparison to latest OpenFIRE GitHub release (or at least latest as of the GUI version).
Add one-click firmware installation/updating from GUI (for both already flashed guns AND RP2040 devices in bootloader mode).""

**Reply by SeongGino** [reply]
...Why are you just repeating what I already had typed out?

Yes, I'm perfectly aware of what I myself have typed out, I do *not* need to be reminded of that. You're not helping your case.

You can help by simply sending a draft MR--it would be more productive on both of our ends rather than filling my inbox with things that comes across as condescending.

Full disclosure: *I am not in a good place right now as it is. Please do not make it worse.*
